modules/bidResponseFilter/index.js

```diff
@@ -0,0 +1,39 @@
+import { config } from '../../src/config.js';
+import { getHook } from '../../src/hook.js';
+
+export const MODULE_NAME = 'bidResponseFilter';
+export const PUBLISHER_FILTER_REJECTION_REASON = 'PUBLISHER_FILTER_REJECTION_REASON';
+export const BID_CATEGORY_REJECTION_REASON = 'BID_CATEGORY_REJECTION_REASON';
+export const BID_ADV_DOMAINS_REJECTION_REASON = 'BID_ADV_DOMAINS_REJECTION_REASON';
+export const BID_ATTR_REJECTION_REASON = 'BID_ATTR_REJECTION_REASON';
+
+function init() {
+  getHook('addBidResponse').before(addBidResponseHook);
+};
+
+export function addBidResponseHook(next, adUnitCode, bid, reject) {
+  const { bcat = [], badv = [], battr = [] } = config.getAnyConfig('ortb2') || {};
+  const moduleConfig = config.getConfig(MODULE_NAME);
+  
+  const catConfig = moduleConfig?.cat || { enforce: true, block_unknown_cat: true };
```

Comment:
if I 
```
setConfig({bidResponseFilter: {cat: {block_unknown_cat: false}}})
```
my expectation would be that `bcat` would still be enforced (e.g., `enforce` defaults to `true`), but I think this would disable it.